### DISCUSSION TITLE
Add overall score for a day

### DISCUSSION THREAD

**Discussion by anishbekalkar** [iSoron/uhabits#1274] [Feature Requests] [answered]
Not sure if this has been requested before but having an (optional) overall score for each day to measure how well a user has been achieving their habits would be pretty helpful. 

An example would be a 100% score for the day if all the habits have been achieved, 50% if half the habits have been reached, and so on. It could be considered as a metahabit, with its own metrics, etc.

**Comment by hiqua** [comment, marked as answer]
Kind of: https://github.com/iSoron/uhabits/discussions/1203
